python/CMakeLists.txt

```diff
@@ -140,6 +141,20 @@ set(MERGE_DYNAMIC_MODULE_FILES
 python_module(TARGET_NAME revng-merge-dynamic WHEEL revng_internal MODULE_FILES
               ${MERGE_DYNAMIC_MODULE_FILES})
 
+#
+# Install revng.scripting
```

Comment:
Let's remove "scripting" as a term entirely. This is either a "client" or a simply a "project".
@mrjackv what do you suggest?

Reply:
`project` seems a bit misleading, I think it can cause confusion (esp. in the docs) with other present or future concepts (e.g. when we talk about multi-binary we often talked about "the project directory")

`client` heavily implies that there's a server, which sometimes there is, but it's a bit dissonant in the context of `cli` stuff

Maybe `wrapper`?

Reply:
Do we need this namespace at all?
This is not "scripting" or "wrapper". This is a first class citizen and the main way users will interact with revng.
I'd be happy to have `revng.profile` and `revng.project`.

Reply:
The files can be moved wherever we want, however doing so would (IMHO) break a bit the current layout:
```
revng.model                 # Model wrapper
revng.pipeline_description  # Pipeline description wrapper
revng.tupletree             # Tuple tree facilities
revng.profile               # Scripting
revng.project               # Scripting
```

Reply:
What do you dislike about it specifically?

There is no concept of "scripting", this should be considered the first and foremost way of using rev.ng, not just "scripting".

If anything, we could consider pushing some other things down into `revng.support`.

Reply:
> What do you dislike about it specifically?

My take is that, it makes things simpler to explain to the user if you can say "only look in `revng.scripting`, all the infra to interact with revng is there" rather than "If you want to interact with revng use `revng.project`, unless you want to use the local daemon or the Hub, at that point you have to go through `revng.profile`"

Reply:
> all the infra to interact with revng is there

You still need `revng.model`.

> If you want to interact with revng use revng.project

I don't think anyone will spawn the daemon by hand and connect manually.
They will almost always go through a profile.

We can push `project` into `revng.profile.project` if you like.